backend/handlers/users/usercredit.go

```diff
@@ -0,0 +1,64 @@
+package usershandlers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"socialpredict/setup"
+	"socialpredict/util"
+
+	"github.com/gorilla/mux"
+	"gorm.io/gorm"
+)
+
+// appConfig holds the loaded application configuration accessible within the package
+var appConfig *setup.EconomicConfig
+
+func init() {
+	var err error
+	appConfig, err = setup.LoadEconomicsConfig()
+	if err != nil {
+		log.Fatalf("Failed to load configuration: %v", err)
+	}
+}
+
+type UserCreditResponse struct {
+	Credit int `json:"credit"`
+}
```

Comment:
I might just call this `UserCredit` given how it's being used (wrapper for an int), and that it may be used in a response but it isn't intrinsically a response.